lib/global.js

```diff
@@ -1,14 +1,3 @@
-var createTorrent = require('create-torrent')
-var wrtc = require('wrtc')
+var wrtc = require('electron-webrtc')()
 
 global.WRTC = wrtc
-
-global.WEBTORRENT_ANNOUNCE = createTorrent.announceList
```

Comment:
> Really, it's not a restriction. If you don't define WEBTORRENT_ANNOUNCE , it doesn't add any trackers at all.

Sorry, with "restriction" I always meant filtering the list of already defined trackers - not the actual logic. Be aware of the reason using `WEBTORRENT_ANNOUNCE`...